packages/terra-menu/src/MenuItem.jsx

```diff
@@ -82,6 +86,14 @@ class MenuItem extends React.Component {
     }
   }
 
+  onKeyUp() {
```

Comment:
Lets rename this to wrapOnKeyUp.

We should apply the same logic as wrapOnKeyDown for the case a customer passes an onKeyUp attributes to the component.